Formula/xctesthtmlreport.rb

```diff
@@ -0,0 +1,19 @@
+class Xctesthtmlreport < Formula
+  desc "Xcode-like HTML report for Unit and UI Tests"
+  homepage "https://github.com/XCTestHTMLReport/XCTestHTMLReport"
+  url "https://github.com/XCTestHTMLReport/XCTestHTMLReport/archive/refs/tags/2.2.2.tar.gz"
+  sha256 "1d7159e4e66feae6194bec2303690e91db17e2593b597ac0db357942020bf4a9"
+  license "MIT"
+  head "https://github.com/XCTestHTMLReport/XCTestHTMLReport.git", branch: "main"
+
+  depends_on xcode: "13.0"
```

Comment:
```suggestion
  depends_on xcode: "13.0"
  uses_from_macos "swift"
```
Is Xcode/Swift needed at runtime?

Reply:
Yes, the tool calls `xcrun`, which is installed with Xcode Command Line Tools. Swift isn't required unless building from source - should it still be listed here?

Reply:
Does it not need a swift runtime?